### PR TITLE
use osm data for actual building height

### PR DESCRIPTION
**Pull request by michaelkirk**
Rather than a random building height, we can often pull real building height from OSM:

Screen Shot 2020-09-21 at 3 50 08 PM

I found that the reallllly tall buildings looked a little too dominate in downtown areas, so I compressed their height a a bit. 

Here's how it would look without that height compression - WDYT?:

Screen Shot 2020-09-21 at 3 56 15 PM

Note that because this adds a field to Building, it requires re-importing map data.

### CODE REVIEW

File: map_model/src/objects/building.rs [hunk 1]
```diff
@@ -26,6 +26,7 @@ impl fmt::Display for BuildingID {
 pub struct Building {
     pub id: BuildingID,
     pub polygon: Polygon,
+    pub levels: f64,
```
Comment (dabreegster) [inline map_model/src/objects/building.rs:29]: I was curious if we could get away with `usize`, but https://taginfo.openstreetmap.org/keys/building%3Alevels#values shows fractional values.

Net file size change from this PR is 3MB, nice!

**Review by dabreegster — COMMENTED** [review]
LGTM!

I found a crash loading the udistrict, NE isometric view, caused by the `Line::must_new` in `DrawBuilding`. Unrelated to this PR, but just FYI. I was going to see how the few giant towers in that area looked relative to everything else.

I have all the data ready to upload. I can send you `data/MANIFEST.txt` and we can coordinate a push, or I can try adding a file to the PR branch.

> Here's how it would look without that height compression - WDYT?:

I think compression is worth it. If the goal is to visualize different building sizes and call out differences, compression achieves it without covering up as much stuff.

**Comment by michaelkirk** [comment]
> I have all the data ready to upload. I can send you data/MANIFEST.txt and we can coordinate a push, or I can try adding a file to the PR branch.


I think you should be able to push to this branch. Give it a try!

Screen Shot 2020-09-22 at 10 10 56 AM

**Comment by dabreegster** [comment]
> I think you should be able to push to this branch. Give it a try!

For the sake of time, I just pushed a separate commit. I successfully pushed to https://github.com/dabreegster/abstreet/compare/michaelkirk-mkirk/building-height?expand=1, but it looks like I needed to clone your repo instead. I'll figure out this workflow eventually. >_<

**Pull request merged**